src/Components/Books/Books.module.css

```diff
@@ -1,4 +1,273 @@
-* {
-  margin: 0;
-  padding: 0;
+@import url('https://fonts.googleapis.com/css2?family=Dangrek&family=Roboto+Slab:wght@200;300;400;500;600;700;800&display=swap');
+
+body {
+  background-color: #f4f5f9;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 6rem;
+}
+
+.Bookcontainer {
+  display: flex;
+  height: 10.625rem;
+  border-radius: 4px;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: 12px auto;
+}
+
+.list {
+  height: 33rem;
+  overflow-y: scroll;
+  margin-top: 40px;
```

Comment:
- [ ] Great job 👏 . Kindly remove the `scrollbar` as it is not included in the original design 🙏 

![image](https://user-images.githubusercontent.com/74814780/201356133-71b312ca-6026-4962-a884-d30728f5199e.png)